cynic-parser/src/parser/schema.lalrpop

```diff
@@ -345,29 +346,99 @@ WrappingType: WrappingType = {
 }
 
 Value: ValueId = {
-    "$" <name:Name> => ast.value(ValueRecord::Variable(name)),
-    <int:IntegerLiteral> => ast.value(ValueRecord::Int(int.parse().unwrap())),
-    <float:FloatLiteral> => ast.value(ValueRecord::Float(float.parse().unwrap())),
-    <start:@L> <s:StringLiteral> =>? {
+    <record:ValueRecord> => {
+        ast.values.value(record)
+    }
+}
+
+ValueRecord: ValueRecord = {
+    <start:@L> "$" <name:Name> <end:@R> => {
+        ValueRecord {
+            span: Span::new(start, end),
+            kind: ValueKind::Variable(values::ids::StringId::from_type_system_id(name))
+        }
+    },
```

Comment:
This should be handled differently since this is schema & variables don't exist there...

Reply:
Handled in https://github.com/obmarg/cynic/pull/1057